doc/apiv3.apib

```diff
@@ -157,12 +154,330 @@ he is allowed to see his own time entries in projects where the costs module is
+                    {
+                        "href": "/api/v3/work_packages/1/cost_entries"
+                    }
+                },
+                "total": 1,
+                "count": 1,
+                "_type": "Collection",
+                "_embedded":
+                {
+                    "elements": [
+                        {
+                            "_type": "CostEntry",
+                            "_links": {
+                                "self": {
+                                    "href": "/api/v3/cost_entries/1"
+                                },
+                                "project": {
+                                    "href": "/api/v3/projects/1"
+                                },
+                                "costType": {
+                                    "href": "/api/v3/cost_types/1"
+                                },
+                                "user": {
+                                    "href": "/api/v3/users/1"
+                                },
+                                "workPackage": {
+                                    "href": "/api/v3/work_packages/1"
+                                }
+                            },
+                            "id": 1,
+                            "spentUnits": 3.14,
+                            "spentOn": "2015-03-31",
+                            "createdAt": "2015-03-31T08:51:20Z",
+                            "updatedAt": "2015-03-31T08:51:20Z"
+                        }
+                    ]
+                }
+            }
+
+## list Cost Entries of a work package [GET]
+
++ Parameters
+    + id (required, integer, `1`) ... work package id
+
++ Response 200 (application/hal+json)
+
+    [Cost Entries by work package][]
+
++ Response 403 (application/hal+json)
+
+    Returned if the client does not have sufficient permissions.
+
+    **Required permission:** view cost entries **or** view own cost entries (on work package's project)
+
+    *Note that you will only receive this error, if you are at least allowed to see the corresponding work package.*
+
+    + Body
+
+            {
+                "_type": "Error",
+                "errorIdentifier": "urn:openproject-org:api:v3:errors:MissingPermission",
+                "message": "You are not allowed to see the cost entries of this work package."
+            }
+
++ Response 404 (application/hal+json)
+
+    Returned if the work package does not exist or the client does not have sufficient permissions
+    to see it.
+
+    **Required permission:** view work package
+
+    *Note: A client without sufficient permissions shall not be able to test for the existence of a work package.
+    That's why a 404 is returned here, even if a 403 might be more appropriate.*
+
+    + Body
+
+            {
+                "_type": "Error",
+                "errorIdentifier": "urn:openproject-org:api:v3:errors:NotFound",
+                "message": "The specified version does not exist."
```

Comment:
Are you sure about the wording here? I'd expect ``work package``  or ``cost entry`` instead of ``version``.